### DISCUSSION TITLE
Create a power on/off and record script every 15 minutes.

### DISCUSSION THREAD

**Discussion by AMBARCAPOOR** [gopro/labs#404] [Q&A] [answered]
I am trying to create a script where the camera will power on, shoot one frame in auto and power down every XX minutes ( let's say 7 minutes). I am having trouble understanding how to best execute this with the commands available in the QR app. 

I want to 

Power on after 6am
Timelapse Photo or Video
Shoot one frame
Power off 
Repeat until 7pm
Repeat Cycle

I would like to use this to record the growth of a hummingbird nest, the laying of eggs, the hatchlings and their flight from the nest so it will probably go over a few weeks. 

And if I need to change a SD card inbetween shots is that possible to do? Or to change a bttery or do I need to restart or reflash the QR code?

If someone has the time I would truly appreciate if you could tell me how you selected the commands so I can learn and build my own variations in the future. Thank you.

Gopro Hero 9 with QR code installed.

**Comment by dnewman-gpsw** [comment, marked as answer]
A script like this will do it: mPdP>05:59<19:00!420SQ~!06:00S!1R

You can cut and past this into the QRControl app.

This was generated from https://gopro.github.io/labs/control/construction/

**Reply by AMBARCAPOOR** [reply]
Thank you! If you have a moment could you break down the command line so I can learn what it means as well. I very much appreciate your time and contribution. Thank you.

**Reply by dnewman-gpsw** [reply]
```
mP - mode Photo
dP - default Photo (for next boot)
>05:59<19:00 - If(time > 5:59am and <7pm)
  !420SQ   -  Sleep for 420 seconds, and start a capture Quickly 
~  -  else
  !06:00S   - Sleep until 6am.
!1R    -  in 1 second Repeat it all
```

**Reply by AMBARCAPOOR** [reply]
great. thank you!
